nixos/modules/services/monitoring/netdata.nix

```diff
@@ -122,9 +122,20 @@ in {
             "error log" = "syslog";
           };
         '';
-        };
+      };
+
+      enableAnalyticsReporting = mkOption {
+        type = types.bool;
+        default = true;
```

Comment:
I think we should default to `false` regardless of what upstream's default behavior is.

This tracking may be anonymous to Netdata Inc., but Google Analytics (or whatever service they'll use) still needs to collect user related data which may even be a legal issue in the EU.

IMO we shouldn't have services in nixpkgs which track users for the benefit of some third party.

Reply:
I'd tend to agree, but I didn't really want to introduce an opinionated choice like that in absence of existing policy (unless that exists and I missed it).

People in the #nixos IRC channel also seemed to be of the opinion it should default to false, too, so I'll change it unless there's some objection.